### ISSUE TITLE
Owner Onboarding Pre spec

### ISSUE THREAD

**Issue by taracw** [taracw/taraproj#1]
- [x] Whiteboard onboarding and follow up process
- [x] Write down emails that don't exist yet in the app ( incomplete boat etc)
- [x] Read through onboarding script
- [x] Admin for rehab?
- [x] Emails for follow up?
- [ ] How to onboard owners that still need it but don't fit into rehab?
- [ ] Market warm ups = "seasonal re-onboarding"
- [x] ~~-Get Laura to do rehab round 3~~
- [x] ~~New google form for onboarding~~

**Comment by taracw** [comment]
**Laura:**
- [x] Upcoming bkg rehab script
- [x] No Upcoming bkg rehab script
- [x] Macro for No Upcoming bkg rehab
- [x] Macro for Upcoming bkg rehab
- [ ] Onboarding 2.0 Typeform
- [x] Rehab round 3
- [x] Onboarding packet
- [x] Review process tree

**Comment by taracw** [comment]
1. Process of onboarding - Scripts
2. Process of finding owners who haven't been onboarded - Boatmaster
3. How the product can onboard the rest - Emails

**Comment by taracw** [comment]
### **Part 2: Process of finding owners that haven't been onboarded - Boatmaster**

**Boatmaster section 1: Onboarding**
- owners that have not been onboarded/ this is their first boat
- boats approved within the last 3 months
- boats that have not been onboarded
- boats that are not marked as follow up
- Boat =  appears_on_site = true 

**Follow up**
Owner's marked as follow up had an onboard call attempted
These owners will receive:
- 1 drip email after 5 days, to reinitiate onboarding
- A 2nd drip email 10 days later, to reinitiate onboarding.

If not onboarded, we won't be removing their boats from the site. Some owners may not need or want a call with us, so they need to be allowed the opportunity to succeed on their own.
The catch all will occur after 15 inquiries, when they end up in rehab.

---

**Boatmaster section 2: Owners that have had zero rentals:**
- owners that have had at least 15 inquiries
- owners that have not had a booking approved, concluded, aboard, ashore, disputed
- Not marked as rehab call or rehab follow up
- Boat =  appears_on_site = true 

**Boatmaster section 3: Owners that have a low approval ratio (LAR)**
- At least 15 INQ
- Approval ratio < 5%
- ??? Owners that don't have a booking approved upcoming???
- Boat =  appears_on_site = true 

Show rehab call  assigned via RB1 or RB2/LAR on user page as well as date
Show rehab follow up as assigned via RB1 or RB2/LAR on user page as well as date

Not sure if even need follow up tag. Could use "boat must appear on site" to help filter this out. But how to mark as rehab if nah....

**Comment by taracw** [comment]
### **Part 3: How the product can onboard the rest - Emails**
1. The first time an owner receives the "welcome to the Boatbound owners club" email, this will include some owner onboard information and the owner onboarding video: welcome to boatbound and setting up your listing.
2. The first time an owner receives the "message owner/ book now" email, this will include some owner onboard information and the owner onboarding video: responding to inquiries.
3. The first time that the owner receives the "owner booking approved" email, this will include some owner onboard information and the owner onboarding video: pre and post rental inspection.

tl;dr: making certain emails smarter, that are the first of their kind.

**Comment by taracw** [comment]
### Emails not in the app
- Owner booking approved email
- Welcome to the Boatbound Owner's club
- Incomplete Listing Drips
- Putting lighthouse content in the dashboard and in the boat management page.  

Chris's notes
- insurance
- fuel
- message responsiveness
- renter verification (you choose who rents etc)
- security deposit/ payment
